classifiers/_template/_template_func/config.py

```diff
@@ -1,10 +1,10 @@
-from util.configs import build_classifier_function_config
+from util.configs import build_extractor_function_config
```

Comment:
is this the correct one? shouldnt it be classifier?

Reply:
Yes! Fixed it.